content/overview/workflow/edit-overview.md

```diff
@@ -6,37 +6,53 @@ weight: 20
 
 ![edit-overview](./edit-overview/edit-overview.png#w100)
 
-1. _Scene-referred_ modules are intended to process pixel values that are proportional to the amount of light collected by the camera at the scene. The dynamic range of an image in the scene-referred section of the pixelpipe is often larger than that of the display medium. 
+1. The [**_lens correction_**](../../module-reference/processing-modules/lens-correction.md) module can optionally be enabled at the start, to correct problems such as barrel/pincushion distortion, vignetting and chromatic aberration. This module will try to automatically detect and correct for your camera and lens combination. Sometimes it may need some help to select the right camera and/or lens.
 
-2. At some point in the pixelpipe, these pixel values are compressed by a tone mapping module into a smaller dynamic range more suitable for display on a monitor or a print.
+2. The [**_exposure_**](../../module-reference/processing-modules/exposure.md) module is adjusted next, in order to set brightness for the main subject in your image. This then allows to you see the image sufficiently well to proceed with the subsequent steps. Note that shadows and highlights may be too dark or bright, or maybe even clipped. Don't worry about that at this stage; just set the exposure of your subject to an appropriate level, and the shadows and highlights will be dealt with later. 
 
-3. The remaining modules operate in this non-linear _display-referred_ section of the pixelpipe to produce the final output image.
+   By default, the exposure module adds +0.5 EV of exposure. Some camera models may need a bit more exposure by default, and an [_auto-apply preset_](../../darkroom/processing-modules/presets.md) can be used to override this value. The _compensate camera exposure_ setting is selected by default, to take into consideration cases where the camera's exposure compensation dial was used to "Expose To The Right" (ETTR). See the [_prerequisites_](../../module-reference/processing-modules/filmic-rgb.md#prerequisites) section of the _filmic rgb_ module for more detail about this workflow. 
 
-Two default workflows are offered in darktable, which define the default starting point for your edits:
+3. The [**_crop and rotate_**](../../../module-reference/processing-modules/crop-rotate.md) module can now be used to straighten the image and frame it correctly. You may also want to use the [_perspective correction_](../../module-reference/processing-modules/perspective-correction.md) module to apply some keystone correction so that vertical and horizontal features in the images are properly aligned. This is the digital equivalent of using a tilt-shift lens, which is commonly used when shooting images of architecture.
```

Comment:
*crop and  rotate* is deprecated and *perspective correction* has been renamed to *rotate and perspective*